upgrade/scripts/upgrade/prepare-assets.sh

```diff
@@ -81,6 +79,9 @@ fi
 
 CSM_REL_NAME="csm-${CSM_RELEASE}"
 CSM_ARTI_DIR="/etc/cray/upgrade/csm/${CSM_REL_NAME}/tarball/${CSM_REL_NAME}"
+
+locOfScript=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${locOfScript}/../common/upgrade-state.sh"
```

Comment:
I don't think this line is necessary

Reply:
I just moved it down from earlier in the file. I assumed it was there because it was needed, but if it isn't, I can remove it.

Reply:
You are right that it is needed. But line 85 does the same think I think and provides a specific node. This is line 85 `. "${locOfScript}/../common/ncn-common.sh" "$(hostname)"`. I don't think you would need both of 84 and 85, correct?

Reply:
Oh, I didn't realize that ncn-common.sh also included upgrade-state. I see that it does, though. Ok, I can remove that line. Happy to eliminate redundancy.

Reply:
Wait, I was very wrong, sorry. You were right initially. I that they were the same file for some reason